index.js

```diff
@@ -10,6 +10,7 @@ const {loggerMiddleware, Logger} = require('@first-lego-league/ms-logger')
 
 const logger = Logger()
 const crudRouter = require('./routers/crudRouter').getRouter
+const { initImagesFolder } = require('./logic/imagesLogic')
```

Comment:
Seems like the index should not be familiar with the logic... I think this initiation, beside it needing to be async, shuold accour in the router